tsdb/engine/tsm1/engine.go

```diff
@@ -2197,7 +2211,7 @@ func (e *Engine) compact(wg *sync.WaitGroup) {
 
 // compactHiPriorityLevel kicks off compactions using the high priority policy. It returns
 // true if the compaction was started
-func (e *Engine) compactHiPriorityLevel(grp CompactionGroup, level int, fast bool, wg *sync.WaitGroup) bool {
+func (e *Engine) compactHiPriorityLevel(grp CompactionGroup, level int, fast bool, wg *sync.WaitGroup, pointsPerBlock int) bool {
```

Comment:
I like passing `pointsPerBlock` in, but I'd like to see `pointsPerBlock` after `fast`. That way the parameters that impact how the compaction is performed are together.